opentelemetry-sdk/setup.cfg

```diff
@@ -45,6 +45,7 @@ include_package_data = True
 install_requires =
     opentelemetry-api == 1.7.1
     opentelemetry-semantic-conventions == 0.26b1
+    opentelemetry-proto == 1.7.1
```

Comment:
We need to just make a separate enum for aggregation temporality in Python to avoid this dependency. This pulls in protobuf.

Reply:
Right, fixing...

Reply:
Removed